cli/cli.ts

```diff
@@ -2335,7 +2336,12 @@ enum BuildOption {
 
 export function serviceAsync(parsed: commandParser.ParsedCommand) {
     let fn = "built/response.json"
-    return mainPkg.serviceAsync(parsed.arguments[0])
```

Comment:
I guess this one could go altogether. It was for testing the webworker interface. Not critical either way.